test/local/DSL/priceCondition.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai'
+import { priceCondition, Token, UniV3Twap } from '@brink-sdk'
+
+describe('priceCondition', function () {
+
+  const USDC_ADDRESS = '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'
+  const DAI_ADDRESS = '0x6b175474e89094c44da98b954eedeac495271d0f'
+
+  it('should return requireUint256LowerBound primitive when operator is <', function () {
+    const primitives = priceCondition({
+      type: 'price',
```

Comment:
@mikec I do not know if we really need to add this argument. we do not use it. the fact that the type is of priceArgument should be enough, no?

Reply:
type param is used here https://github.com/brinktrade/brink-sdk/blob/0569e697f8d4824dea72bf6301933e5582dc407e/src/internal/intentSegmentArgsToOrderArgs.ts#L63

Reply:
Ok fair enough.

Yes I understand it's definitely needed in the DSL since JSON is not typed 👍